x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/profile/ProfileDocument.java

```diff
@@ -96,7 +90,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
 
     public Subject subject() {
         return new Subject(
-            new User(user.username, user.roles.toArray(String[]::new), user.fullName, user.email, Map.of(), user.active),
+            new User(user.username, user.roles.toArray(String[]::new), user.fullName, user.email, Map.of(), true),
```

Comment:
Good question. The answer is No. Only enabled user can authenticate which is a pre-requisite for creating profile document. Technically, it does not matter here because the subsequent usage does not concern about whether this value is true or false.